### ISSUE TITLE
Melonds looks bad

### ISSUE THREAD

**Issue by g-k-m** [melonDS-emu/melonDS#1685]
I did some comparisons of melonds vs desmume on windows 10, both latest versions 0.9.5 and 0.9.13, using yu-gi-oh 5D's WC 2011

Melonds settings
![image](https://github.com/melonDS-emu/melonDS/assets/37273303/7eeeff80-2737-41fa-a728-698558559c23)
Screen filtering off, window size 3x, microphone input off, audio bitrate 16x, everything else default settings

Desmume settings
![image](https://github.com/melonDS-emu/melonDS/assets/37273303/a7fde017-0dfa-4798-abef-999466e3be84)
Display method opengl, filter off, window size 3x, advanced bus-level timing off, dynamic compiler on with 100 block size (for extra performance to get to 6x gpu scaling, 7x drops below 60 fps, melonds easily had 60 fps on 16x so no need for cpu emulation), everything else default settings

Game "waiting to press button" screen:

![melonds](https://github.com/melonDS-emu/melonDS/assets/37273303/7e803f43-9097-4c43-b24f-4d0b6f54b614)
Melonds

![desmume1](https://github.com/melonDS-emu/melonDS/assets/37273303/ebc7c347-3677-4381-965e-504a07fcdb17)
Desmume

bottom window looks the same in both pretty much (at least i can't notice a difference), however top window looks miles better on desmume

3d during-movement screen:

![melonds2](https://github.com/melonDS-emu/melonDS/assets/37273303/51bac568-28cc-48b4-b88d-a0922d91b304)
Melonds

![desmume](https://github.com/melonDS-emu/melonDS/assets/37273303/04676284-8294-4a4b-8243-1a63356cac05)
Desmume

I could not catch exactly the same frame on both because it was during movement, but it's about as same-y as it gets. As you can see, despite melonds running in 16x native resolution while desmume only running in 6x, somehow desmume looks miles better, again

I don't know why melonds looks so bad in comparison, but i just thought i'd leave this here for discussion on how to make melonds look better, because right now, there is no reason to use it over desmume, the difference in quality is just night and day

**Comment by RSDuck** [comment]
it's just display capture.

**Comment by ashgx** [comment]
0.9.13？=0.9.3？

**Comment by patataofcourse** [comment]
0.9.13 as in the latest desmume version

**Comment by Nixel1324** [comment]
Actually, I think the difference here is texture upscaling, a feature MelonDS doesn't have yet. This has nothing to do with the resolution the final image is being rendered at, it's just a filter applied to the textures. With the 3D scene screenshot, you can tell the geometry is perfectly sharp/not pixelated, but the texture applied to the model is upscaled with some algorithm on Desmume.

**Comment by RSDuck** [comment]
you are right, idk what I thought while saying display capture (probably not at all). Yeah this is basically just a very badly written feature request for texture hi res resampling with something like hqx.
